packages/suite/src/components/suite/AppNavigation/index.tsx

```diff
@@ -1,22 +1,108 @@
 import React, { useRef, useLayoutEffect, useState } from 'react';
 import styled, { css } from 'styled-components';
 import { variables, IconProps, useTheme, Button, Icon, Dropdown } from '@trezor/components';
-import { Translation } from '@suite-components';
-import { useSelector, useActions } from '@suite-hooks';
-import * as routerActions from '@suite-actions/routerActions';
+import { AccountFormCloseButton } from '@suite-components';
+import { isInView } from '../AppNavigationPanel';
+import { useSelector } from '@suite-hooks';
 import { Route } from '@suite-types';
+import AccountStickyContent from '../AccountStickyContent';
+import { MAX_WIDTH, MAX_WIDTH_WALLET_CONTENT } from '@suite-constants/layout';
 
 const { FONT_WEIGHT, FONT_SIZE } = variables;
 
 const SECONDARY_MENU_BUTTON_MARGIN = '12px';
 
-const Wrapper = styled.div<{ settingsWrapper: boolean | undefined }>`
+const Wrapper = styled.div<{ value: boolean; subRoute: boolean | undefined }>`
+    width: 100%;
+    z-index: 2;
     display: flex;
+    background: ${props => props.theme.BG_LIGHT_GREY};
+    justify-content: center;
+
+    ${props =>
+        props.subRoute &&
+        props.value &&
+        `
+            position: sticky;
+            top: 0;
+            left: 0;
+            right: 0;
+            bottom: 0;
+            height: 0;
+            transform: translate(0, -71px);
+        `}
```

Comment:
would be nice to tag it with `css` from styled-components for syntax highlighting, autocomplition...
```suggestion
        css`
            position: sticky;
            top: 0;
            left: 0;
            right: 0;
            bottom: 0;
            height: 0;
            transform: translate(0, -71px);
        `}
```

Reply:
same on other places in this file

Reply:
331a45e053a53457e46506ccff18f29dd5f5b5fc